tiledb/libtiledb.pyx

```diff
@@ -3301,6 +3301,43 @@ cdef class ArraySchema(object):
         print("\n")
         return
 
+    def serialize(self, format='JSON'):
+        """Returns a JSON (as string) representation of the array schema"""
+        cdef tiledb_serialization_type_t serialization_type
+        if format == 'JSON':
+            serialization_type = TILEDB_JSON
+        else:
+            raise Exception("Capnp unimplemented")
+
+
+        cdef tiledb_buffer_t* buff
+        check_error(self.ctx, tiledb_buffer_alloc(self.ctx.ptr, &buff))
```

Comment:
Not necessary to alloc a buffer here -- https://github.com/TileDB-Inc/TileDB/blob/dev/tiledb/sm/c_api/tiledb.cc#L5077 does it automatically